src/sentry/api/endpoints/relay/details.py

```diff
@@ -9,6 +9,7 @@
 
 class RelayDetailsEndpoint(Endpoint):
     permission_classes = (SuperuserPermission,)
+    enforce_rate_limit = True
```

Comment:
`enforce_rate_limit = True` means we'll enable the rate limit on this endpoint as soon as it deploys. We want to use the global `enforce_rate_limit` setting instead.